src/sentry/lang/javascript/processor.py

```diff
@@ -166,9 +167,9 @@ def discover_sourcemap(result):
         # We want to scan each line sequentially, and the last one found wins
         # This behavior is undocumented, but matches what Chrome and Firefox do.
         for line in possibilities:
-            if line[:21] in ("//# sourceMappingURL=", "//@ sourceMappingURL="):
+            if line[:21] in (b"//# sourceMappingURL=", b"//@ sourceMappingURL="):
                 # We want everything AFTER the indicator, which is 21 chars long
-                sourcemap = line[21:].rstrip()
+                sourcemap = line[21:].rstrip().decode("utf-8")
```

Comment:
See my other comment, I think we should respect the encoding gotten from the request result. It would probably be `utf-8` for 99% of the time but when we hit that 1%, we'd have head aches.